source/features/quick-pr-diff-options.tsx

```diff
@@ -127,8 +128,19 @@ void features.add(import.meta.url, {
 }, {
 	include: [
 		pageDetect.isSingleCommit,
+	],
+	shortcuts: {
+		'd w': 'Show/hide whitespaces in diffs',
+	},
```

Comment:
Do we have to copy the shortcut info in every init object? Seems like having it once would be enough.

Reply:
The issue is that `shortcuts` is only considered if the feature applies to the current page. This key is used to generate the `?` view